azerothcore/azerothcore-wotlk
# Fix(DB/creature_loot_template): Emblem of Triumph

## CHANGES PROPOSED:
Match MinCount and MaxCount for [Emblem of Triumph] in boss loot.

(Credit to @Voxstrasza for catching my mistake in old PR.)
## ISSUES ADDRESSED:
Some bosses drop 1-2 Emblems of Triumph when it should be either 1-1 or 2-2.

## TESTS PERFORMED:
Kel'thuzad 10-man: [2x Triumph](https://youtu.be/gx-R_dq8L5Q?t=455)
Sapphiron 10-man: [1x Triumph](https://youtu.be/fxuDBq0nh1Q?t=594)
XT-002 25-man: [Normal: 1x Triumph](https://youtu.be/EIXYaSEuTTI?t=488), [Hard: 1x Triumph](https://youtu.be/NFDRR7gkNfc?t=501)
Auriaya 25-man: [1x Triumph](https://youtu.be/xQUGItPWGMs?t=256)

## HOW TO TEST THE CHANGES:
Check before and after
```sql
SELECT `Name`, `Item`, `Mincount`, `Maxcount` FROM `creature_loot_template` INNER JOIN `creature_template` ON `creature_loot_template`.`entry` = `creature_template`.`lootid` WHERE `item` = 47241 AND `creature_loot_template`.`entry` IN (15989,33694,33724,33885,34175,15990);
```

If you want to test in-game:
- Set raid mode 10-player
Sapphiron ```.go cre 133932```
Kel'thuzad ```.go cre 133933```

- Set raid mode 25-player
XT-002 ```.go cre 136054```
Auriaya ```.go cre 137496```
Razorscale ```.go cre 137611```
Stormcaller Brundir ```.go cre 136316```

## Target branch(es):


## How to test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here in the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR
